src/test/java/com/arturfrimu/exercisesback/service/ExerciseGenerationServiceImplTest.java

```diff
@@ -23,20 +23,22 @@
 class ExerciseGenerationServiceImplTest {
 
     @Autowired
-    private ExerciseGenerationServiceImpl exerciseGenerationService;
+    private ExerciseManagementService exerciseManagementService;
+    @Autowired
+    private FindPercentageExerciseService findPercentageExerciseService;
 
     @BeforeEach
     public void setup() {
         MockitoAnnotations.openMocks(this);
-        exerciseGenerationService.clear();
+        exerciseManagementService.clear();
     }
 
     @MethodSource("getPercentageArgumentsProvider")
     @ParameterizedTest
     void getPercentage(final Map<UUID, Exercise> map, PercentageResponse percentageResponse) {
-        exerciseGenerationService.put(map);
+        exerciseManagementService.put(map);
 
-        PercentageResponse percentage = exerciseGenerationService.getPercentage();
+        PercentageResponse percentage = findPercentageExerciseService.getPercentage();
```

Comment:
Clasa se incepe cu find, numele metodei se incepe cu get

Ele sunt sinonime

Foloseste acelasi verb si in clasa si in metoda

Si daca clasa se numeste FindExerciseService e cam redundand sa mai numesti si metoda findExercise, deja din denumirea clasei se intelege ce anume ai de gand sa cauti

Si de asta poti metoda s-o chemi doar find